src/main/java/org/eclipse/microprofile/starter/core/validation/PackageNameValidator.java

```diff
@@ -23,17 +23,16 @@
 package org.eclipse.microprofile.starter.core.validation;
 
 import javax.enterprise.context.ApplicationScoped;
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 @ApplicationScoped
 public class PackageNameValidator {
 
-    private static final String VALID_REGEX = "^(?:\\w+|\\w+\\.\\w+)+$";
+    public static final String VALID_REGEX = "^(?:\\w+|\\w+[\\.-]\\w+)+$";
+    public static final int MAX_LENGTH = 100;
```

Comment:
@Karm Any reason for the max length?

Reply:
@rdebusscher While I was looking at the validation, it seemed to me we should cap this at some limit, otherwise the limit is server's default which is something like 10MB per request (headers+body) for Thorntail (Undertow) IIRC. I am much more a httpd guy than a Java/Rest Easy one though, so there might be much more elegant way of doing this...